src/data/courseDependencies.ts

```diff
@@ -0,0 +1,6 @@
+import dependencies from './courseDependenciesData'
```

Comment:
moved this into the `src` folder and copied over the `js` file instead of using JSON so it is the same and "just works" with a cut and paste.